style.css

```diff
@@ -0,0 +1 @@
+h1 { color: brown; }
```

Comment:
**[REQUIRED CHANGES]**

*Please Note: I'd utilize this inline feature to add all the changes you need to implement, Kindly bear in mind*


*LINTER*
- As it's a requirement for all projects in microverse, please kindly add the linter file and they should be set up correctly with no linter error.

*README*

- Please, kindly update your readme by personalizing the content in the headers and removing the headers that are not relevant to the project to make your readme look more professional.
Example:
1. Getting started: In this header, you should add some instructions on how a user can have your repo locally on their machine. (hot to clone the repo, how to move into the cloned repo), 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.
3. - Please, kindly replace the `Major languages` with `HTML`, and `Frameworks` with `CSS`, and remove the `Technologies` to make the readme well arranged to look more professional.
4. Please, kindly make the authors singular and update the contents with your social media links.
5. Please, kindly remove or update the live demo link to link to the live demo version of your project as now, it's a different link.

*.gitignore*
- Kindly add the `test.md` file, the `node_modules/` directory, and remove the `.github` you've added.